package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "vnc-keymaps",
+  "version": "0.0.1",
+  "type": "module",
+  "description": "Keymps for VNC",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    "import": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "require": {
+      "types": "./dist/index.d.cts",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "files": [
+    "./dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com:kubevirt-ui/vnc-keymaps.git"
+  },
+  "author": "Radoslaw Szwajkowski <rszwajko@redhat.com>",
+  "license": "Apache-2.0",
```

Comment:
why not MIT ?

Reply:
MIT may be a better choice but some more research is needed.